src/slackers/server.py

```diff
@@ -52,6 +52,10 @@
     if action.callback_id:
         event_type = f"{action.type}:{action.callback_id}"
         emit(actions, event_type, action)
+    elif action.view:
```

Comment:
Use `if` instead of `elif` to emit both on action as well as view when we have both